src/cgw_connection_processor.rs

```diff
@@ -40,6 +45,12 @@ enum CGWConnectionState {
     ClosedGracefully,
 }
 
+#[derive(Debug, PartialEq)]
+enum CGWUCentralMessageProcessorState {
+    Idle,
+    Pending,
```

Comment:
Most likely you meant something like ResultPending?
It's not clear what it is pending for :)